.github/workflows/semgrep.yml

```
@@ -0,0 +1,48 @@
+name: Semgrep
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)